task_14/js/task7.js

```diff
@@ -0,0 +1,11 @@
+let p = document.getElementById('id');
+let x = prompt('Введите число');
+let temp;
+let result = 1;
+while (x > 0){
+    temp = x % 10;
+    result = result * temp;
```

Comment:
You can make this code less
result *= x % 10;

Reply:
done